extensions/trace-propagators/src/main/java/io/opentelemetry/extension/trace/propagation/B3PropagatorInjectorMultipleHeaders.java

```diff
@@ -26,6 +26,12 @@
 
     String sampled = spanContext.isSampled() ? Common.TRUE_INT : Common.FALSE_INT;
 
+    String debug = context.get(B3Propagator.DEBUG_CONTEXT_KEY);
+    if (!StringUtils.isNullOrEmpty(debug) && debug.contentEquals(Common.TRUE_INT)) {
+      setter.set(carrier, B3Propagator.DEBUG_HEADER, Common.TRUE_INT);
```

Comment:
Should we use the new constant here, `MULTI_HEADER_DEBUG`?

Reply:
yeh, have changed it/tidied it up.